spec/rails_helper.rb

```diff
@@ -67,3 +69,12 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
+
+
+Capybara.configure do |config|
+  # https://www.rubydoc.info/github/jnicklas/capybara/Capybara.configure
+  config.automatic_label_click = true
+  config.default_max_wait_time = 1
+  config.ignore_hidden_elements = false # Fix: matchers not finding elements
+  config.javascript_driver = :selenium_chrome_headless
+end
```

Comment:
can you move this into `spec/support/capybara.rb` - have been trying to keep config and customisation for different components in separate files, for organisational reasons